packages/core/src/autoscaling/snapshotter.ts

```diff
@@ -472,13 +467,9 @@ export class Snapshotter {
     protected async _ensureCorrectMaxMemory() {
         if (this.maxMemoryBytes) return;
         const { totalBytes } = await this._getMemoryInfo();
-        if (isAtHome()) {
-            this.maxMemoryBytes = totalBytes;
-        } else {
-            this.maxMemoryBytes = Math.ceil(totalBytes / 4);
-            // NOTE: Log as AutoscaledPool, so that users are not confused what "Snapshotter" is
-            this.log.info(`Setting max memory of this run to ${Math.round(this.maxMemoryBytes / 1024 / 1024)} MB. Use the ${ENV_VARS.MEMORY_MBYTES} environment variable to override it.`); // eslint-disable-line max-len
-        }
+        this.maxMemoryBytes = Math.ceil(totalBytes / 4);
```

Comment:
same here, i will try to deal with the `isAtHome` thingy in the other PR, but if we remove it, it will be easy to forget about it (and will break the platform usage)